ar_exercise/exercise.rb

```diff
@@ -51,13 +59,27 @@ class Apartment < ActiveRecord::Base
 # Hint, the following methods will help: `new`, `create`, `save`, `uddate`, `destroy`
 
 # Create 3 new apartments, and save them to the DB
+northwest =Apartment.create("1133 15th Street NW", 6000, 1, 3)
```

Comment:
Good practice is to explicitly match attributes with values (e.g., `Apartment.create(address: "1133 15th St NW")`)